packages/tools/fetch-tool/src/fluidFetchSnapshot.ts

```diff
@@ -56,11 +57,11 @@ function isFetchedTree(fetchedData: IFetchedData): fetchedData is IFetchedTree {
     return "patched" in fetchedData;
 }
 
-const blobCache = new Map<string, Promise<string>>();
-let blobCachePrevious = new Map<string, Promise<string>>();
-let blobCacheCurrent = new Map<string, Promise<string>>();
+const blobCache = new Map<string, Promise<ArrayBufferLike>>();
+let blobCachePrevious = new Map<string, Promise<ArrayBufferLike>>();
+let blobCacheCurrent = new Map<string, Promise<ArrayBufferLike>>();
 
-function fetchBlobs(prefix: string,
+async function fetchBlobs(prefix: string,
```

Comment:
I think async should be removed, as intention of this function is to kick off a bunch of async work, but not actually blob.
Other way to say it - in the past, it was capable to start hundreds of async requests in parallel.
Now we process items why by one due to added awaits. I believe awaits should be removed and we should continue to work with promises.